### PR TITLE
tests: verify `bootupctl status` output is not empty

### PR DESCRIPTION
**Pull request by HuijingHei**
See https://github.com/coreos/bootupd/issues/694

### CODE REVIEW

File: tests/kola/boot/bootupd [hunk 1]
```diff
@@ -22,12 +22,21 @@ case "$(arch)" in
     aarch64|x86_64)
         # on these arches, we always expect state files to exist
         check_state_file=1
+        # aarch64 (and x86_64) uses uefi firmware, should check grub2-efi
+        if ! bootupctl status | grep "grub2-efi"; then
```
Comment (jlebon) [inline tests/kola/boot/bootupd]: Ahh I see, you'd have to make this additionally conditional to check for either grub2-efi-x64 or -aa64 to be able to do a NEVRA check. Hmm, how about we get the EVR for `grub2-common`, which is on both, and then do `grep "grub2-efi-.*-${evr}"`?

File: tests/kola/upgrade/extended/test.sh [hunk 2]
```diff
@@ -145,7 +145,7 @@ ok "Reached version: $version"
 if vereq $version $target_version; then
     ok "Fully upgraded to $target_version"
     # log bootupctl information for inspection where available
-    [ -f /usr/bin/bootupctl ] && /usr/bin/bootupctl status
+    [ -f /usr/bin/bootupctl ] && (/usr/bin/bootupctl status | grep "CoreOS aleph version")
```
Comment (jlebon) [inline tests/kola/upgrade/extended/test.sh]: I think the intent here was to have that output go to the test logs if we ever wanted to inspect it. Maybe to still do that, let's instead do `| tee out.txt` and then as a separate command below do `grep` on that file?

**Review by jlebon — COMMENTED** [review]
Nice! Thank you so much for tackling this.

File: tests/kola/boot/bootupd (hunk 1 above)
Comment (HuijingHei) [reply tests/kola/boot/bootupd]: SGTM, thanks for the suggestion.

**Review by travier — APPROVED** [review]

**Pull request merged**